set.mm

```diff
@@ -417031,45 +417031,46 @@ to v(n-2)=v.  This theorem holds even for k=0, but not for n=2, since
     $( Lemma for ~ clwwlknonclwlknonf1o :  The bijective function of
        ~ clwwlknonclwlknonf1o is the bijective function of ~ clwlknf1oclwwlkn
        restricted to the closed walks with a fixed positive length on a fixed
-       vertex.  (Contributed by AV, 26-May-2022.) $)
+       vertex.  (Contributed by AV, 26-May-2022.)  (Revised by AV,
+       7-Aug-2022.) $)
     clwwlknonclwlknonf1olem $p |- ( ( G e. USPGraph /\ N e. NN )
-          -> F = ( ( c e. { w e. ( ClWalks ` G ) | ( # ` ( 1st ` w ) ) = N }
-                     |-> ( ( 2nd ` c ) substr <. 0 , ( # ` ( 1st ` c ) ) >. ) )
+          -> F = ( ( w e. { c e. ( ClWalks ` G ) | ( # ` ( 1st ` c ) ) = N }
+                     |-> ( ( 2nd ` w ) substr <. 0 , ( # ` ( 1st ` w ) ) >. ) )
                    |` W ) ) $=
-      ( wcel wa cv c1st cfv chash wceq crab c2nd cuspgr cclwlks cc0 cop csubstr
-      cn co cmpt cres wss wi simpl a1i ss2rabdv syl5eqss resmpt syl syl6reqr )
-      CUALDUFLMZHANZOPQPDRZACUBPZSZHNZTPUCVDOPQPUDUEUGZUHFUIZHFVEUHZBUSFVCUJVFV
-      GRUSFVAUCUTTPPGRZMZAVBSVCJUSVIVAAVBVIVAUKUSUTVBLMVAVHULUMUNUOHVCFVEUPUQKU
-      R $.
+      ( wcel wa c1st cfv chash wceq c2nd cc0 csubstr cuspgr cn cclwlks crab cop
+      cv cmpt cres wss ssrab3 a1i weq fveq2 fveq2d eqeq1d fveq1d anbi12d elrab2
+      co simpl simplbiim adantl ssrabdv resmptd opeq2d oveq12d cbvmptv syl6reqr
+      syl6eq ) CUALDUBLMZAHUFZNOZPOZDQZHCUCOZUDZAUFZROZSVQNOZPOZUEZTUSZUGFUHZHF
+      VKROZSVMUEZTUSZUGZBVJWCAFWBUGWGVJAVPFWBVJVNHVOFFVOUIVJVTDQZSVROZGQZMZAVOF
+      JUJUKVKFLZVNVJWLVKVOLVNSWDOZGQZMZVNWKWOAVKVOFAHULZWHVNWJWNWPVTVMDWPVSVLPV
+      QVKNUMUNZUOWPWIWMGWPSVRWDVQVKRUMZUPUOUQJURVNWNUTVAVBVCVDAHFWBWFWPVRWDWAWE
+      TWRWPVTVMSWQVEVFVGVIKVH $.
```

Comment:
Why did you lengthen the proof ?

Reply:
as explained above: to shorten the proof of clwwlknonclwlknonf1o

Reply:
But the statement looks unchanged ?

Reply:
Tje variables changed ( w <-> c )! And this requires some conversion steps (~cbvmptv etc.).

Reply:
Ah, I see... This does not appear in the diff, which only shows the $p-statement, so I thought it was simple alpha-renaming, but there are $e-hypotheses (which do not appear in the diff) which contain `c` and `w` variables and are unchanged, so this is not simple alpha-renaming.